src/inspector/models/text/textsettingsmodel.cpp

```diff
@@ -42,6 +42,7 @@ TextSettingsModel::TextSettingsModel(QObject* parent, IElementRepositoryService*
 {
     setSectionType(InspectorSectionType::SECTION_TEXT);
     setTitle(qtrc("inspector", "Text"));
+    setIsSpecialCharactersInsertionAvailable(isTextEditingStarted());
```

Comment:
I would recommend moving the setting of the value under the isTextEditingChanged().onNotify processing because these things are linked